tests/parm/regional_rrfs_a.nml.IN

```diff
@@ -0,0 +1,314 @@
+&amip_interp_nml
+    data_set = 'reynolds_oi'
+    date_out_of_range = 'climo'
+    interp_oi_sst = .true.
+    no_anom_sst = .false.
+    use_ncep_ice = .false.
+    use_ncep_sst = .true.
+/
+
```

Comment:
`amip_interp_nml` namelist is not used by the model. Should be removed.

Reply:
corrected